snippets/hooks.json

```diff
@@ -6290,5 +5997,305 @@
       "Alter the links displayed at the top of the view edit form."
     ],
     "scope": "php"
+  },
+  "hook_aggregator_fetcher_info_alter": {
+    "prefix": "hook_aggregator_fetcher_info_alter",
+    "body": [
+      "/**",
+      " * Implements hook_aggregator_fetcher_info_alter().",
+      " */",
+      "function ${1:${TM_FILENAME_BASE:hook}}_aggregator_fetcher_info_alter(array &$info) {",
+      "  $0",
+      "}"
+    ],
+    "description": [
+      "hook_aggregator_fetcher_info_alter",
+      "Drupal 9+",
+      "",
+      "Perform alterations on the available fetchers."
+    ],
+    "scope": "php"
+  },
+  "hook_aggregator_parser_info_alter": {
+    "prefix": "hook_aggregator_parser_info_alter",
+    "body": [
+      "/**",
+      " * Implements hook_aggregator_parser_info_alter().",
+      " */",
+      "function ${1:${TM_FILENAME_BASE:hook}}_aggregator_parser_info_alter(array &$info) {",
+      "  $0",
+      "}"
+    ],
+    "description": [
+      "hook_aggregator_parser_info_alter",
+      "Drupal 9+",
+      "",
+      "Perform alterations on the available parsers."
+    ],
+    "scope": "php"
+  },
+  "hook_aggregator_processor_info_alter": {
+    "prefix": "hook_aggregator_processor_info_alter",
+    "body": [
+      "/**",
+      " * Implements hook_aggregator_processor_info_alter().",
+      " */",
+      "function ${1:${TM_FILENAME_BASE:hook}}_aggregator_processor_info_alter(array &$info) {",
+      "  $0",
+      "}"
+    ],
+    "description": [
+      "hook_aggregator_processor_info_alter",
+      "Drupal 9+",
+      "",
+      "Perform alterations on the available processors."
+    ],
+    "scope": "php"
+  },
+  "hook_ckeditor_plugin_info_alter": {
```

Comment:
Oh, here's the D9.5 only hooks - need to make sure the above removals are matched with these additions.

Reply:
Might be a good idea to sort the hooks alphabetically so they major version additions don't re-arrange the whole file in the future.